lib/freeathome.js

```diff
@@ -117,7 +134,11 @@ module.exports = class FreeAtHomeApi {
                     const ioObject = serialNumber + '.' + channel + '.' + datapoint;
                     this.adapter.setObjectNotExists(ioObject, {
                         type: 'state',
-                        common: {},
+                        common: {
+                            name: 'RAW value',
+                            read: true,
+                            write: true
```

Comment:
Same here: Type could be missing

Reply:
This one is tough, I assume it could be of different types, depending on the device. I don't know what to specify here.